bigbluebutton-html5/imports/ui/components/chat/container.jsx

```diff
@@ -117,6 +122,20 @@ const ChatContainer = (props) => {
   const chatName = participants?.filter((user) => user.id !== Auth.userID)[0]?.name;
   const title = chatName ? intl.formatMessage(intlMessages.titlePrivate, { 0: chatName}) : intl.formatMessage(intlMessages.titlePublic);
 
+  const { users } = usingUsersContext;
+  let partnerIsLoggedOut = false;
+
+  if(!isPublicChat){
+    const idUser = participants?.filter((user) => user.id !== Auth.userID)[0]?.id;
+    partnerIsLoggedOut = (users[idUser]?.loggedOut || users[idUser]?.ejected) ? true : false;
+
+    if (partnerIsLoggedOut) {
+      isChatLocked = true;
```

Comment:
as a good practice, I recommend not reassigning the values of the props

Reply:
updated in 448d1b0